site/assets/scss/_variables.scss

```diff
@@ -20,4 +20,10 @@ $bd-callout-variants: info, warning, danger !default;
   --bd-teal-rgb: #{to-rgb($teal-500)};
   --docsearch-primary-color: var(--bd-violet);
   --docsearch-logo-color: var(--bd-violet);
+  --bd-sidebar-link-bg: rgba(var(--bd-violet-rgb), .1);
+}
+
+[data-bs-theme="dark"] {
+  --bd-violet: #{mix($bd-violet, $white, 75%)};
```

Comment:
In dark mode, the contrast ratio (3.91:1) doesn't seem correct for regular texts.

![Screenshot 2022-11-24 at 17 39 40](https://user-images.githubusercontent.com/17381666/203833846-06d3babd-b4ee-4eb3-a409-febd9279aa9f.png)


![Screenshot 2022-11-24 at 17 44 50](https://user-images.githubusercontent.com/17381666/203833862-5cef61a2-e862-420e-beba-85239bd2e4f1.png)

If the values found with the color picker are correct: foreground `#f0e9ff` and background `#9461fb`